scripts/ci/perf_test.sh

```diff
@@ -1,45 +1,26 @@
 set -ex -o pipefail
 
-case "${TEST_FORMAT}" in
-  http|flux-http)
-    ;;
-  *)
-    >&2 echo "Unknown query format: ${TEST_FORMAT}"
-    exit 1
-    ;;
-esac
+source vars.sh
 
-# get latest ubuntu 20.04 ami for us-west-2
-ami_id=$(aws --region us-west-2 ssm get-parameters --names /aws/service/canonical/ubuntu/server/20.04/stable/current/amd64/hvm/ebs-gp2/ami-id --query 'Parameters[0].[Value]' --output text)
+# get latest ubuntu 21.10 ami for us-west-2
+ami_id=$(aws --region us-west-2 ssm get-parameters --names /aws/service/canonical/ubuntu/server/21.10/stable/current/amd64/hvm/ebs-gp2/ami-id --query 'Parameters[0].[Value]' --output text)
 
 # launch ec2 instance
-instance_type="r5.2xlarge"
 datestring=$(date +%Y%m%d)
 instance_info=$(aws --region us-west-2 ec2 run-instances \
   --image-id $ami_id \
-  --instance-type $instance_type \
+  --instance-type $DATA_I_TYPE \
   --block-device-mappings DeviceName=/dev/sda1,Ebs={VolumeSize=200} \
   --key-name circleci-oss-test \
   --security-group-ids sg-03004366a38eccc97 \
   --subnet-id subnet-0c079d746f27ede5e \
-  --tag-specifications "ResourceType=instance,Tags=[{Key=Name,Value=oss-perftest-$datestring-${CIRCLE_BRANCH}-${CIRCLE_SHA1}-${TEST_FORMAT}}]")
+  --tag-specifications "ResourceType=instance,Tags=[{Key=Name,Value=oss-perftest-$datestring-${CIRCLE_BRANCH}-${CIRCLE_SHA1}}]")
 
 # get instance info
 ec2_instance_id=$(echo $instance_info | jq -r .Instances[].InstanceId)
+echo "export EC2_INSTANCE_ID=$ec2_instance_id" >> vars.sh
 
-# pull down the latest influx_tools
```

Comment:
Instead of pulling influx_tools to the circleci machine and then SCP'ing to the performance test machine, just pull it directly to the test machine.